### DISCUSSION TITLE
Send paper plane icon for Flex Webchat

### DISCUSSION THREAD

**Discussion by Porlin** [twilio-labs/paste#2162] [Request: Icon] [answered]
Hi There,

We are working on Flex Webchat and I'm using this icon for the message input field to send a message. Please see attached 
<img width="335" alt="Screenshot 2022-01-27 at 12 39 23" src="https://user-images.githubusercontent.com/49287701/151360646-ce2efb79-f756-453e-ac8b-28e24a126744.png">
<img width="43" alt="Screenshot 2022-01-27 at 12 39 29" src="https://user-images.githubusercontent.com/49287701/151360655-655546a4-2be1-42ea-9d06-9de3fba7bdfb.png">

I noticed that Paste have a similar icon that is used for Marketing Campaigns - see here 
 
<img width="61" alt="Screenshot 2022-01-27 at 12 21 06" src="https://user-images.githubusercontent.com/49287701/151360748-82a91474-3dce-4e31-a266-8da423a39728.png">

Can we please request a new icon for the Paper plane Send icon?

**Comment by shleewhite** [comment]
Hi @Porlin! 👋🏽 This is definitely an icon that belongs in Paste! We will work on adding this in the next couple sprints. Will follow up again when it is available. Do you have a timeline for when you need this icon?

**Reply by Porlin** [reply]
Hi @shleewhite , thanks for getting back to me. Regarding timeline, unfortunately the icon is a super urgent one for us. The team have finished the build and need to release in Feb. Is there a way design systems can expedite this one for us?

**Reply by shleewhite** [reply]
Unfortunately we can't right now - if you are able, you can contribute the icon to Paste yourself. We have resources on the process here: https://paste.twilio.design/introduction/contributing/icons/.

**Comment by Porlin** [comment]
Thanks @shleewhite will keep you updated on how I get on with the icon.

**Comment by Porlin** [comment]
Hi @shleewhite just sharing some icons taken from streamline. Streamline - Send Email 3 is the one we want to go with. Sharing here

<img width="1445" alt="Screenshot 2022-02-02 at 09 38 08" src="https://user-images.githubusercontent.com/49287701/152129248-0cacb353-c6eb-487e-92eb-1ac34b4b4d3a.png">

<img width="728" alt="Screenshot 2022-02-02 at 09 38 40" src="https://user-images.githubusercontent.com/49287701/152129190-91394971-ec94-4744-9ecb-9e7efa96170e.png">

**Reply by gloriliale** [reply]
Hi @Porlin! Thanks for sending these over! We were wondering, would you be open to rotating the icon (similar to the Slack send icon), that way it’s more differentiated from the marketing campaigns logo?

**Comment by Porlin** [comment]
@gloriliale Sure, I'm open to it and so is the team. Good observation and suggestion. I have some examples here 

<img width="711" alt="Screenshot 2022-02-03 at 13 55 18" src="https://user-images.githubusercontent.com/49287701/152412834-6763a634-7a84-4996-a1ed-ca50c58b4324.png">

Option 2 is the preferred. 

For next steps I'll add this to the figma library listed in your steps?

**Comment by Porlin** [comment]
Hey @gloriliale and @shleewhite 
I have made a branch for the Send icon in the Paste components here: https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/branch/GLcr5vNnWGwfaCRM87DK8N/Paste-Components

The icon is ready for your review here:
![action_Send](https://user-images.githubusercontent.com/49287701/152550735-360dd992-b00a-47ff-b1ec-f1615b3741d3.svg)

Please let me know next steps. Thanks for your help on this.

**Reply by gloriliale** [reply]
Hi @Porlin! We should have this merged in very soon! I'll update here when it's all set.

**Comment by Porlin** [comment]
Awesome! Thanks @gloriliale

**Comment by gloriliale** [comment, marked as answer]
Hi @Porlin! The icon has been merged in and released, you should be able to update your icons package and find it there! 🎉

**Comment by Porlin** [comment]
That's wonderful! Thanks @gloriliale
